public/javascripts/i18n.js

```
@@ -0,0 +1,1080 @@
+    //     %z/%Z  - Timezone offset (+0545)
+    //
+    I18n.strftime = function (date, format, options) {
+        var options = this.lookup('date', options);
+        var meridianOptions = I18n.meridian();
+        if (!options) {
+            options = {};
+        }
+
+        options = this.prepareOptions(options, DATE);
+
+        if (isNaN(date.getTime())) {
```

Biome (1.9.4)

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)